python/flask-example/README.md

```diff
@@ -1,13 +1,27 @@
 # Sample Flask Application 
 Python Flask example inspired by previous bottle Example from the [IBM developerWorks wiki](https://www.ibm.com/developerworks/community/wikis/home?lang=en#!/wiki/IBM%20i%20Technology%20Updates/page/Sample%20web%20application%20with%20Python)
 
-# Installing requisites
+This example can be used with 2 different ways of accessing DB2 on IBM i.
+
+##  Using *ibm_db* and *itoolkit*
+
+### Installing requisites for itoolkit access
  - **ibm_db & itoolkit:** `yum install python3-ibm_db python3-itoolkit`
  - **flask:** `pip3 install flask`
 
-# Starting the server 
+### Starting the server 
 python3 ./sample.py
 
+##  Using *odbc*
+
+### Installing requisites for itoolkit access
+ - **pyodbc** `pip3 install pyodbc`
```

Comment:
Should note that when running `pip3 installing pyodbc` pip will compile pyodbc from source therefore the required compilers and source files must be available for the compile to succeed.

```bash
$ yum groupinstall "Development Tools"

$ yum install python3-devel unixODBC-devel
```

Reply:
We ship python3-pyodbc, so we should recommend users use that instead.

Reply:
Nice! I did not notice we shipped `pyodbc` as an rpm. Yes should definitely use yum to install it instead.

```bash
$ yum install python3-odbc
```

Reply:
OK, so I am updating the line to say
### Installing requisites for itoolkit access
 - **pyodbc** `yum install python3-pyodbc`